requirements/common_constraints.txt

```diff
@@ -33,3 +34,10 @@ elasticsearch<7.14.0
 # So we need to pin it globally, for now.
 # Ticket for unpinning: https://github.com/openedx/edx-lint/issues/407
 importlib-metadata<7
+
+# Cause: https://github.com/openedx/event-tracking/pull/290
+# event-tracking 2.4.1 upgrades to pymongo 4.4.0 which is not supported on edx-platform.
+# We will pin event-tracking to do not break existing installations
+# This can be unpinned once https://github.com/openedx/edx-platform/issues/34586
```

Comment:
I would add a comment on this ticket saying exactly this so the work doesn't slip through the cracks!